frontends/p4/specialize.cpp

```diff
@@ -16,26 +16,31 @@ limitations under the License.
 
 #include "specialize.h"
 
-#include <iostream>
-
+#include "frontends/common/constantFolding.h"
+#include "frontends/common/resolveReferences/referenceMap.h"
```

Comment:
Did you use a tool to identify the missing includes? If so, what did you use? I'd potentially like to run that elsewhere.

Reply:
One of the possible tools to use is include-what-you-use: https://github.com/include-what-you-use/include-what-you-use, though it might require some settings for large codebases.

Here I am using include cleaner from clang-tidy / clangd: https://clangd.llvm.org/design/include-cleaner (https://clang.llvm.org/extra/clang-tidy/checks/misc/include-cleaner.html)